includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -457,7 +477,7 @@ private function get_wordpress_stable_version() {
 				if ( isset( $body['offers'] ) && ! empty( $body['offers'] ) ) {
 					$latest_release = reset( $body['offers'] );
 
-					$version = $latest_release['new_bundled'];
+					$version = $latest_release['current'];
```

Comment:
I stand corrected, `current` is actually the correct field to use, not `new_bundled`.